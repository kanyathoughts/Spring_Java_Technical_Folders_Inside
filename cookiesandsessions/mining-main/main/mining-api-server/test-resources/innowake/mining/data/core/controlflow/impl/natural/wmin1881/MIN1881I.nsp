DEFINE DATA
LOCAL USING Z1069--A
LOCAL USING Z1069--L
LOCAL
01 #FILE-RECORD         (A20)
01 REDEFINE #FILE-RECORD 
   02 #FILE-RECORD-SORT (A10)
END-DEFINE
*
INCLUDE Z1069--C
*
DEFINE WORK FILE 1 'WF-TEST'
SET CONTROL '+'
*
UPLOAD PC FILE 1 #FILE-RECORD
	WRITE(1) NOTITLE NOHDR '=' #FILE-RECORD
END-ALL
*
SORT #FILE-RECORD-SORT USING #FILE-RECORD
	WRITE(1) NOTITLE NOHDR '=' #FILE-RECORD
END-SORT
*
CLOSE PC FILE 1
*
END
