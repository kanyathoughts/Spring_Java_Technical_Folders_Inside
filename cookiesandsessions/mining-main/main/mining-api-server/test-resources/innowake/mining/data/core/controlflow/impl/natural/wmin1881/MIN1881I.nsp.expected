 #2 DefineDataStmt
 #15 IncludeStmt, FROM={TestEntryPoint}, TO={#16}
 #16 DefineWorkFileStmt, FROM={#15}, TO={#18}
 #18 SetControlStmt, FROM={#16}, TO={#20}
 #20 UploadPcFileStmt, Label: UPLOAD PC FILE 1 #FILE-RECORD, FROM={#18, #25}, TO={#25(LOOP BODY), #33(LOOP END)}
  #24 EveryRecord
   #25 WRITE(1) NOTITLE NOHDR '=' #FILE-RECORD, FROM={#20(LOOP BODY)}, TO={#20}
   #32 EndAllMarkStmt
 #33 SortStmt, Label: SORT #FILE-RECORD-SORT USING #FILE-RECORD, FROM={#20(LOOP END), #38}, TO={#38(LOOP BODY), #45(LOOP END)}
  #38 WRITE(1) NOTITLE NOHDR '=' #FILE-RECORD, FROM={#33(LOOP BODY)}, TO={#33}
 #45 ClosePcFileStmt, FROM={#33(LOOP END)}, TO={#46}
 #46 EndStmt, FROM={#45}, TO={TestReturnPoint}
